src/test/java/org/dcsa/api_validator/bkg/v1/shipments/testGetShipmentByCarrierBookingReference.java

```diff
@@ -0,0 +1,93 @@
+package org.dcsa.api_validator.bkg.v1.shipments;
+
+import io.restassured.path.json.JsonPath;
+import org.apache.http.HttpStatus;
+import org.dcsa.api_validator.conf.Configuration;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.dcsa.api_validator.bkg.v1.BookingTestConfiguration.*;
+import static org.dcsa.api_validator.bkg.v1.BookingTestConfiguration.SHIPMENT_CARRIERBOOKINGREFERENCE_PATH;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+
+import static org.dcsa.api_validator.bkg.v1.BookingTestConfiguration.JSON_SCHEMA_VALIDATOR;
+import static org.hamcrest.Matchers.*;
+
+/*
+ * Tests related to the GET /shipments/{carrierBookingReference} endpoint
+ */
+
+public class testGetShipmentByCarrierBookingReference {
+
+    private List<String> carrierBookingReferenceListForShipments;
+
+    @BeforeClass
+    public void getAllCarrierBookingReferencesForShipments() {
+        this.carrierBookingReferenceListForShipments =
+                JsonPath.from(
+                                given()
+                                        .auth()
+                                        .oauth2(Configuration.accessToken)
```

Comment:
.auth()
 .oauth2(Configuration.accessToken)
 
 This can be removed,  we do not need to an accesstoken for dev/test env.